src/sentry/static/sentry/app/views/settings/incidentRules/metricField.tsx

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import {css} from '@emotion/core';
+
+import FormField from 'app/views/settings/components/forms/formField';
+import {t, tct} from 'app/locale';
+import {QueryField} from 'app/views/eventsV2/table/queryField';
+import {generateFieldOptions} from 'app/views/eventsV2/utils';
+import {FieldValueKind} from 'app/views/eventsV2/table/types';
+import {Organization} from 'app/types';
+import space from 'app/styles/space';
+import FormModel from 'app/views/settings/components/forms/model';
+import Button from 'app/components/button';
+import Tooltip from 'app/components/tooltip';
+import {
+  explodeFieldString,
+  generateFieldAsString,
+  AggregationKey,
+  FieldKey,
+  AGGREGATIONS,
+  FIELDS,
+} from 'app/utils/discover/fields';
+
+import {Dataset} from './types';
+
+type Props = Omit<FormField['props'], 'children' | 'help'> & {
+  organization: Organization;
+};
+
+const cannedAggregates = [
+  {
+    match: /^count\(\)/,
+    name: 'Number of errors',
+    validDataset: [Dataset.ERRORS],
+    default: 'count()',
+  },
+  {
+    match: /^count_unique\(user(\.id)?\)/,
+    name: 'Users affected',
+    validDataset: [Dataset.ERRORS],
+    default: 'count_unique(user)',
+  },
+  {
+    match: /^(p[0-9]{2}|percentile\(transaction\.duration,[^)]+\))/,
```

Comment:
There is also a p100 which doesn't match this pattern.

Reply:
ah, good catch